paper-time-picker.html

```diff
@@ -375,6 +383,11 @@
           this.rawValue = rawValue;
         },
         _updateTime: function(hour, minute, second) {
+          if (hour === undefined || minute === undefined ||
```

Comment:
Also need to account for enableSeconds being undefined, which you may as well just pass into this function while you're at it : )